src/styles/utils/variables.scss

```diff
@@ -0,0 +1,15 @@
+$dark-blue: #060b35;
+$grey: #616070;
+$light-blue: #00acdc;
+$white: #fff;
+$light-grey:#f3f3f3;
+$font-size-small: 10px;
+$font-size-regular: 12px;
+$font-size-big: 14px;
+$font-size-biggest: 16px;
+$font-weight-regular: 400;
+$font-weight-medium: 500;
+$font-weight-bold: 700;
+$card-width: 200px;
+$card-gapX: 48px;
+$card-gapY: 51px;
```

Comment:
this one is tricky - you need to have 50px from nav
and 46px between cards vertically